dns/controller_test.go

```diff
@@ -0,0 +1,104 @@
+package dns
+
+import (
+	"testing"
+
+	"time"
+
+	"fmt"
+
+	"github.com/sky-uk/feed/k8s"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+const smallWaitTime = time.Millisecond * 50
+
+type fakeClient struct {
```

Comment:
I don't think you can easily share test code (by intention). We'd have to create non test code to share and depend on testify in it.